otter/convergence/composition.py

```diff
@@ -22,12 +22,12 @@ def tenant_is_enabled(tenant_id, get_config_value):
         "convergence-tenants" portion of the configuration file.
     :param callable get_config_value: config key -> config value.
     """
-    enabled_tenant_ids = get_config_value("convergence-tenants")
```

Comment:
We can always have additive deploys by first deploying "non-convergence-tenants" config change followed by this PR change. Would those steps be comfortable?